### PR TITLE
hey spaceman i think you got the wrong vault

### PR DESCRIPTION
**Pull request by Iamgoofball**
the zombie club is 2 z levels down

thanks goons for the fitness machines sounds and sprites

### CODE REVIEW

**Comment by KazeEspada** [comment]
GET SWOLE BRO

RD:"Mining, wheres my minerals?"
Miner:"Busy getting swole, brah"

**Comment by TehZombehz** [comment]
This has been needed forever.

**Comment by Archie700** [comment]
>in lavaland

FUCK YOU

**Comment by Iamgoofball** [comment]
fuck you duckhead

**Comment by Aloraydrel** [comment]
oh yea smartass?

**Comment by Durkel** [comment]
I'll show you whos the boss of this gym.

**Comment by duncathan** [comment]
This is good to merge during the freeze, as it only adds a new map

**Comment by KingDairy** [comment]
>duckhead

it's duckman, actually

**Comment by Cheridan** [comment]
please put goon assets in their own /goon folder

**Comment by optimumtact** [comment]
DYEL son?

@RemieRichards where's our exercycles wanna buff my cardio

**Comment by RemieRichards** [comment]
nobody ever gave me sprites ;_;

File: _maps/templates/lavaland_gym.dmm [hunk 1]
```diff
@@ -0,0 +1,52 @@
+"a" = (/turf/simulated/mineral/volcanic/lava_land_surface,/area/lavaland/surface/outdoors)
+"b" = (/turf/simulated/floor/plating/asteroid/basalt/lava_land_surface,/area/lavaland/surface/outdoors)
+"c" = (/turf/simulated/wall/r_wall,/area/ruin/powered)
+"d" = (/obj/structure/closet,/turf/simulated/floor/plasteel/freezer{baseturf = /turf/simulated/floor/plating/lava/smooth},/area/ruin/powered)
+"e" = (/obj/structure/closet,/mob/living/simple_animal/hostile/zombie,/turf/simulated/floor/plasteel/freezer{baseturf = /turf/simulated/floor/plating/lava/smooth},/area/ruin/powered)
+"f" = (/obj/structure/punching_bag,/turf/simulated/floor/plasteel/freezer{baseturf = /turf/simulated/floor/plating/lava/smooth},/area/ruin/powered)
+"g" = (/turf/simulated/floor/plasteel/freezer{baseturf = /turf/simulated/floor/plating/lava/smooth},/area/ruin/powered)
+"h" = (/obj/structure/window{icon_state = "window"; dir = 1},/obj/item/weapon/shard{icon_state = "medium"},/mob/living/simple_animal/hostile/zombie{name = "Boss of This Gym"},/turf/simulated/floor/plasteel/freezer{baseturf = /turf/simulated/floor/plating/lava/smooth},/area/ruin/powered)
```
Comment (KorPhaeron) [inline _maps/templates/lavaland_gym.dmm]: This might not be a good idea because zombie outbreaks.

**Comment by KorPhaeron** [comment]
What is the deep lore of a gym being in hell?

**Comment by KorPhaeron** [comment]
I also can't help but notice you failed to put a wrestling belt in the gym

**Comment by PKPenguin321** [comment]
What's the deep lore of hell being right next to a space station

**Comment by KorPhaeron** [comment]
@PKPenguin321 

They're mining there duh.

**Comment by optimumtact** [comment]
As the dorf's say, if you aint mining hell, you aint living.

**Comment by Archie700** [comment]
>no dead mob named leatherman

missed opportunity

**Comment by Iamgoofball** [comment]
@Archie700 check again

File: _maps/templates/lavaland_gym.dmm [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+"m" = (/obj/structure/window{icon_state = "window"; dir = 8},/turf/simulated/floor/plasteel/freezer{baseturf = /turf/simulated/floor/plating/lava/smooth},/area/ruin/powered)
+"n" = (/obj/structure/window{icon_state = "window"; dir = 4},/turf/simulated/floor/plasteel/freezer{baseturf = /turf/simulated/floor/plating/lava/smooth},/area/ruin/powered)
+"o" = (/obj/structure/stacklifter,/turf/simulated/floor/plasteel/freezer{baseturf = /turf/simulated/floor/plating/lava/smooth},/area/ruin/powered)
+"p" = (/obj/structure/window,/obj/structure/window{icon_state = "window"; dir = 8},/obj/effect/decal/cleanable/blood,/turf/simulated/floor/plasteel/freezer{baseturf = /turf/simulated/floor/plating/lava/smooth},/area/ruin/powered)
+"q" = (/obj/structure/window,/turf/simulated/floor/plasteel/freezer{baseturf = /turf/simulated/floor/plating/lava/smooth},/area/ruin/powered)
+"r" = (/obj/structure/window,/obj/item/weapon/shard,/mob/living/simple_animal/hostile/zombie{name = "Leatherhead"},/turf/simulated/floor/plasteel/freezer{baseturf = /turf/simulated/floor/plating/lava/smooth},/area/ruin/powered)
```
Comment (KazeEspada) [inline _maps/templates/lavaland_gym.dmm]: Leatherhead

**Comment by KingDairy** [comment]
the spelling looks incorrect though
or just make it like borscht where he can spawn as either -man or -head

**Comment by Cheridan** [comment]
assets still need to be put into their own folder

**Comment by Cheridan** [comment]
u did it

**Pull request merged**